app-infrastructure/db/auth/V11__Insert_RAS_Connection.sql

```diff
@@ -0,0 +1,14 @@
+INSERT INTO connection
+(uuid, label, id, subprefix, requiredFields)
+VALUES (
+  UNHEX(REPLACE('9770edc4-1533-4796-9005-5cfb77c0f28d', '-', '')),
```

Comment:
Why not just have it in the expected format instead of the replace?

Reply:
Because I am lazy and it was easier to write the unhex and replace functions.